src/custom/components/LoadingRows/index.tsx

```diff
@@ -0,0 +1,34 @@
+import styled, { keyframes } from 'styled-components/macro'
+
+const loadingAnimation = keyframes`
+  0% {
+    background-position: 100% 50%;
+  }
+  100% {
+    background-position: 0% 50%;
+  }
```

Comment:
just curious, whats this effect. I didn't notice

Reply:
I'm not 100% sure but this might be for the bug that can happen with the shimmer type css animation which I had problem with before. Now I didn't saw it happen so I would keep this.

Reply:
Its a shimmer effect on the table rows while the data is loading
![Screenshot from 2022-06-02 15-19-36](https://user-images.githubusercontent.com/34926005/171638462-049568fe-4986-4ed8-b1f2-2b102f471422.png)